### PR TITLE
feat: added generic gatt dis characteristic uuid

### PR DESCRIPTION
**Pull request by daantimmer**
These UUIDs are generic to BLE.
Also fixed the location of services/network/gap.proto, which should have been services/ble/gap.proto instead

### CODE REVIEW

**Review by EkelmansPh — APPROVED** [review]

File: services/ble/Gatt.hpp [hunk 1]
```diff
@@ -37,6 +37,20 @@ namespace services
         };
     };
 
+    namespace uuid
+    {
+        constexpr static AttAttribute::Uuid16 deviceInformationService{ 0x180A };
```
Comment (richardapeters) [inline services/ble/Gatt.hpp:42]: Please check if constexpr inline is a better choice

**Review by richardapeters — APPROVED** [review]

File: services/ble/Gatt.hpp [hunk 2]
```diff
@@ -37,6 +37,20 @@ namespace services
         };
     };
 
+    namespace uuid
```
Comment (rjaegers) [inline services/ble/Gatt.hpp:40]: @oguzcanoguz I believe these were previously (re)moved. What was the rationale back then, and is that still relevant now?

Comment (oguzcanoguz) [reply services/ble/Gatt.hpp:40]: Rationale was that we shall not put all those standard services in infralib. DIS is one and there can be surely others later.

Comment (magi-arun) [reply services/ble/Gatt.hpp:40]: DIS is generic service and used widely unlike other standard services which are specific for the application, DIS can be part of ble services in infralib.

Comment (daantimmer) [reply services/ble/Gatt.hpp:40]: @rjaegers @oguzcanoguz @magi-arun let me know if this issue can be resolved as won't fix, or if additional work needs to be done?

Comment (magi-arun) [reply services/ble/Gatt.hpp:40]: As DIS is widely used BLE service, agreed to retain the dis uuid  as part of infralib. Closing this comment as won't fix.

**Pull request merged**